src/API_modules/API_functionality.js

```diff
@@ -0,0 +1,40 @@
+import { key } from './API_key.js';
+
+export default class API {
+  constructor() {
+    this.apiName = 'Haider Ahmad';
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: